src/components/swap/GasEstimateTooltip.tsx

```diff
@@ -0,0 +1,61 @@
+import { sendAnalyticsEvent } from '@uniswap/analytics'
+import { InterfaceElementName, SwapEventName } from '@uniswap/analytics-events'
+import { Currency, TradeType } from '@uniswap/sdk-core'
+import { LoadingOpacityContainer } from 'components/Loader/styled'
+import { RowFixed } from 'components/Row'
+import { MouseoverTooltip, TooltipSize } from 'components/Tooltip'
+import { InterfaceTrade } from 'state/routing/types'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { ReactComponent as GasIcon } from '../../assets/images/gas-icon.svg'
+import SwapRoute from './SwapRoute'
+
+const StyledGasIcon = styled(GasIcon)`
+  margin-right: 4px;
+  height: 18px;
+  & > * {
```

Comment:
my css knowledge may be a bit weak but i think a comment like `// style all elements in the svg icon` would be useful - i had to look up what this syntax did